atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -560,6 +561,17 @@ def get_build(self, metadata):
                 else:
                     extra['filesystem_koji_task_id'] = task_id
 
+        help_result = self.workflow.prebuild_results.get(AddHelpPlugin.key)
+        if help_result is not None:
+            try:
+                cgit_url = self.workflow.source.uri.replace('/rpms/', '/cgit/rpms/')
```

Comment:
The requirement is that the metadata can be used to discover the dist-git URL, commit hash, and pathname to the Markdown source file.

It needn't be a single field.

Note that build.source already includes the dist-git URL and commit hash, so I think all that's needed is to include the path within the repository in the build.extra.image map.